#request
int32 address
---
#response
bool level
bool success
